src/document_change_manager.ts

```diff
@@ -390,12 +390,7 @@ export class DocumentChangeManager implements Disposable {
             changeArgs.push([start.line, startBytes, end.line, endBytes, text.split(eol)]);
         }
 
-        const bufTick: number = await this.client.request("nvim_buf_get_changedtick", [bufId]);
-        if (!bufTick) {
```

Comment:
I hesitated slightly here, but in my testing, I don't think it's possible for `bufTick` to return a falsy value (unless it's zero, but based on the log, that doesn't seem to be the intent of the log). If we pass an unknown buffer, it throws.